packages/vats/src/core/chain-behaviors.js

```diff
@@ -304,9 +304,13 @@ export const startTimerService = async ({
   vats: { timer: timerVat },
   consume: { client },
   produce: { chainTimerService: produceTimer },
+  brand: {
+    produce: { timer },
+  },
```

Comment:
`TimerBrand` has a different API from ERTP `Brand`s. Are you sure that no one will try to send `getDisplayInfo()` to all the brands in this list?  I'm suspicious that this should get a different top-level category.

Reply:
I gave it a try (8ebd8b8), but after a couple hours, it's not working and I can't figure out why.

@ivanlei suggested I timebox it. So I think I'm going ahead without it.